### ISSUE TITLE
Total influence

### ISSUE THREAD

**Issue by Csenevirathna** [chathika/MACM#24]
https://github.com/chathika/MACM/blob/7bb917aa9b670c54f68b8c876de4936a74555853/MACM/MACM.py#L219

Check this line again, please. In line 204 and line 215 x.sum() is considered to find the endo influence and exo influence. But in line 219, x.mean() is considered to find the total influence.

**Comment by deamonpog** [comment]
This is calculating mean values of entropy.

**Comment by Csenevirathna** [comment]
Yeah, it is calculating the mean value of entropy. I am not sure, but for
me, I think if you calculate mean entropy there, then you need to calculate
the mean endo and Exo transfer entropy too. Not the total endo or exo
transfer entropies. If you consider the total (summation) endogenous and
exogenous TE, then shouldn't you consider the total entropy too?
That is what I thought. I might be wrong.

Best Chathurani.
